### ISSUE TITLE
Add support for C16, C116, Plus/4 computers

### ISSUE THREAD

**Issue by Esshahn** [nurpax/petmate#79]
Support for those computers should be straight forward:

- different color palette
- different addresses in assembly export
- slightly different BASIC exporter

Good starting point could be to analyze Marq's handling of the plus/4 format in terms of colors.
I would happily provide the BASIC exporter.

![screenshot 2018-08-12 um 13 09 40](https://user-images.githubusercontent.com/434355/44001389-647c149c-9e31-11e8-8581-1c8aca4f3a1e.jpg)

**Comment by nurpax** [comment]
Makes sense for orthogonality but probably will not go intonthe next releasr line. :)

**Comment by Esshahn** [comment]
Some quick observations:
Colors are incremental from top left to bottom right:

![screenshot 2018-08-12 um 20 19 02](https://user-images.githubusercontent.com/434355/44005056-371dca68-9e6d-11e8-864e-48b7d4c551ca.jpg)

Gives values for colram in .c file

![screenshot 2018-08-12 um 20 19 23](https://user-images.githubusercontent.com/434355/44005060-45f12328-9e6d-11e8-9f68-4f8892545394.jpg)

So with changed values for col and screen ram in the BASIC code

````
20 poke 65305,6
30 poke 65301,0
100 for i = 3072 to 3072 + 999
110 read a: poke i,a: next i
120 for i = 2048 to 2048 + 999
130 read a: poke i,a: next i
````

the current BASIC exporter does already work on C16/ Plus/4 (color values are wrong of course)

![screenshot 2018-08-12 um 20 38 15](https://user-images.githubusercontent.com/434355/44005201-dd54a756-9e6f-11e8-841e-e8828445f777.jpg)
![screenshot 2018-08-12 um 20 37 20](https://user-images.githubusercontent.com/434355/44005202-e2ce20b8-9e6f-11e8-96f9-93e765bb400c.jpg)
![screenshot 2018-08-12 um 20 37 29](https://user-images.githubusercontent.com/434355/44005204-edb67b06-9e6f-11e8-8b6b-e67a1298d1b1.jpg)
